src/NextGen/dataRepository.ts

```diff
@@ -0,0 +1,26 @@
+import { Batch } from '@mparticle/event-models';
+import { BaseMPMessage } from './core/baseMPMessage';
+
+export class MParticleDataRepository {
+    public internalState: BaseMPMessage[] = [];
+
+    public insertCommerceEvent(data: BaseMPMessage) {
+        this.internalState.push(data);
+    }
+
+    public insertBreadcrumb(data: BaseMPMessage): void {
+        this.internalState.push(data);
+    }
```

Comment:
I'm curious why `insertBreadcrum` explicitly returns void where `insertCommerceEvent` does not.

Reply:
that was probably just an oversight, i added the suggestion